### ISSUE TITLE
Accessibility: Element not highlighted on focus - Slideshow arrows

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#434]
**Description**
On any carousel that is used, the arrows are not highlighted when they are focused with keyboard navigation. Can be seen in the [Black Excellence: Fashion that Prevails exhibit report](https://my2.siteimprove.com/Inspector/602695/Accessibility/Page?pageId=40593957623&impmd=3CC33A985F387DC37EA87D3D5A2306C4#/Criterion/2.4.7/Check/63).

**Solution**
Need to remove the `outline: 0` declaration in the CSS for this element when it is focused.
